### ISSUE TITLE
Update build and docs to reflect how we deploy MetacatUI now

### ISSUE THREAD

**Issue by amoeba** [NCEAS/metacat#1451]
Related to https://github.com/NCEAS/metacat/issues/1447.

When we switched MetacatUI away from using fragment URLs, we needed to make use of Apache's [`FallbackResource`](https://httpd.apache.org/docs/trunk/mod/mod_dir.html) directive to support arbitrary entry (not from the root) access. i.e., `/metacatui/view/foo`. Currently, if users deploy MetacatUI using default approach we use w/ Metacat builds -- alongside metacat as another Tomcat deployment -- arbitrary entry into MetacatUI doesn't work (you get a Tomcat 404).

@taojing2002 and I have looked at this and Tomcat appears to have a similar config option to `mod_dir`'s `FallbackResource` but we couldn't make it work the same. If we did find a way, it'd involve adding yet another step to the config process which is already long imo.

Deploying MetacatUI directly under Apache is how we're doing all of our own deployments and I think we find it very manageable. Is it possible for us to un-bundle MetacatUI or change how we bundle it (i.e., not as a WAR)? Given that MetacatUI deployed under Tomcat doesn't _fully_ work, I think this'd be a good move. At this point, I'd vote for bundling a plain old ZIP of the latest release of MetacatUI and editing the instructions to tell the user to unzip under their Apache document root.

**Comment by laurenwalker** [comment]
I agree with this suggestion. We should also direct people to the metacatui docs for installation details, which include Apache directives. I need to update the Metacat docs with that.

**Comment by laurenwalker** [comment]
Oh, I'd also like to see the default MetacatUI deployment location at the origin root instead of /metacatui. I don't think anyone will want to deploy a production environment with `metacatui` in the URL.

**Comment by mbjones** [comment]
Those issues seem important.  

Our goal in bundling MetacatUI in Metacat was so that, with a relatively simple drop-in install, someone could have a fully functional repo with default UI, which is what the original Metacat deployment provided. Now that SOLR is independent, and our Apache config is quite detailed, I think it would be best to unbundle the two products, but provide instructions in the Metacat Admin manual as to how to obtain and install MetacatUI alongside Metacat. The Metacat Admin utility might also detect when MetacatUI is properly installed, and if not, provide a link to the instructions.  

I think we should also provide a docker-compose based deployment that allows a one-command mechanism to bring up a site that puts all of the needed components in place (apache, postgres, tomcat, solr, metacat, metacatui, ...). That is covered in a separate ticket in metacat.

**Comment by amoeba** [comment]
Thanks @mbjones. Re:

> ...I think it would be best to unbundle the two products, but provide instructions in the Metacat Admin manual as to how to obtain and install MetacatUI alongside Metacat...

This simplifies things one step further which I like. I'm down.
